pkg/generate/app/pipeline.go

```diff
@@ -33,8 +34,6 @@ func NewImagePipeline(from string, image *ImageRef) (*Pipeline, error) {
 }
 
 func NewBuildPipeline(from string, input *ImageRef, strategy *BuildStrategyRef, source *SourceRef) (*Pipeline, error) {
-	strategy.Base = input
```

Comment:
The base image setup is now handled in `StrategyAndSourceForRepository` by `FromSourceRef`.